pkg/version/windows_versioninfo.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
another thing is some filed like you said (Patch) in our code `OS_GIT_COMMIT` is a string, but in versioninfo is int https://github.com/josephspurrier/goversioninfo/blob/master/goversioninfo.go#L50
I think we need to think how to map our current exist value to the right palaces, if we be much easier if we have those mapping tables as a document.